integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -444,6 +450,172 @@ public void testLivenessProbe() {
+    String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+    assertTrue(assertDoesNotThrow(
+        () -> doesFileExistInPod(domainNamespace, adminServerPodName, dataFileToCheck),
+        String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace)),
+        String.format("did not find file %s in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace));
+
+    // check in admin server pod, the default admin server data file is in default data store
+    String defaultAdminDataFile =
+        "/u01/domains/" + domainUid + "/servers/admin-server/data/store/default/_WLS_ADMIN-SERVER000000.DAT";
+    assertTrue(assertDoesNotThrow(() ->
+            doesFileExistInPod(domainNamespace, adminServerPodName, defaultAdminDataFile),
+        String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+            dataFileToCheck, adminServerPodName, domainNamespace)),
+        String.format("did not find file %s in pod %s in namespace %s",
+            defaultAdminDataFile, adminServerPodName, domainNamespace));
+
+    // check in managed server pod, there is no custom data file for JMS is created
+    for (int i = 1; i <= replicaCount; i++) {
+      for (int j = 1; j <= NUMBER_OF_CLUSTERS_MIIDOMAIN; j++) {
+        String managedServerPodName = domainUid + "-cluster-" + j + "-" + MANAGED_SERVER_NAME_BASE + i;
+        String customDataFile = "/u01/oracle/customFileStore/FILESTORE-0@MANAGED-SERVER" + i + "000000.DAT";
+        assertFalse(assertDoesNotThrow(() ->
+                doesFileExistInPod(domainNamespace, managedServerPodName, customDataFile),
+            String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+                customDataFile, managedServerPodName, domainNamespace)),
+            String.format("found file %s in pod %s in namespace %s, expect not exist",
+                customDataFile, managedServerPodName, domainNamespace));
+
+        String defaultMSDataFile = "/u01/domains/" + domainUid + "/servers/cluster-" + j + "-managed-server" + i
+            + "/data/store/default/_WLS_CLUSTER-" + j + "-MANAGED-SERVER" + i + "000000.DAT";
+        assertTrue(assertDoesNotThrow(() ->
+                doesFileExistInPod(domainNamespace, managedServerPodName, defaultMSDataFile),
+            String.format("exception thrown when checking file %s exists in pod %s in namespace %s",
+                defaultMSDataFile, managedServerPodName, domainNamespace)),
+            String.format("can not find file %s in pod %s in namespace %s",
+                defaultMSDataFile, managedServerPodName, domainNamespace));
+      }
+    }
+  }
+
+  @Test
+  @DisplayName("Test dataHome override in a domain with domain on PV type")
+  public void testDataHomeOverrideDomainOnPV() {
```

Comment:
Move the description to javadoc with complete sentences, as per comments on previous method

Reply:
fixed